ethereum/core/src/main/java/tech/pegasys/teku/core/ForkChoiceUtil.java

```diff
@@ -385,7 +385,15 @@ public static AttestationProcessingResult on_attestation(
     Attestation attestation = validateableAttestation.getAttestation();
 
     return validateOnAttestation(store, attestation, forkChoiceStrategy)
-        .ifSuccessful(() -> indexAndValidateAttestation(validateableAttestation, maybeTargetState))
```

Comment:
Looks like you can cut `indexAndValidateAttestation` altogether now.  Just wondering if we need the call to `attestation.saveCommitteeShufflingSeed(targetState);` that's in that method?

Reply:
Good catch. Added that method call to an ifSuccesful block right after this one.